docs/faq/network/network-configuration.md

```diff
@@ -0,0 +1,56 @@
+
+- [Lighthouse Port Specification](https://lighthouse-book.sigmaprime.io/faq.html?highlight=9000#do-i-need-to-set-up-any-port-mappings)
+- [Prysm Port Specification](https://docs.prylabs.network/docs/prysm-usage/p2p-host-ip#configure-your-firewall)
+- [Geth Port Specification](https://github.com/ethereum/go-ethereum#configuration)
+- [Erigon Port Specification](https://github.com/ledgerwatch/erigon#default-ports-and-firewalls)
+
+### Do the LUKSO mainnet and testnet use different ports?
+
+Mainnet and Testnet will run on the same client ports. If a validator wants to run two clients on the same machine, you must configure custom ports for the second machine.
+
+### Do I need a stable internet connection to run a validator?
+
+Running an EVM Proof of Stake validator node at home requires a stable and reliable internet connection with sufficient network speeds and low latency. A stable internet connection is crucial to ensure optimal performance and maintain the validator's active status within the network. Here's why speeds and latency are essential:
+
+- **Blockchain Synchronization**: A fast and stable internet connection is required to synchronize the blockchain data with the Ethereum network efficiently. Slow download speeds can lead to longer sync times, which may prevent your validator from actively participating in the consensus process, causing missed opportunities for rewards.
+
+- **Attestations and Proposals**: As a validator, your node creates attestations and proposes blocks. A low-latency connection ensures that your attestations and proposals reach the network promptly, increasing the likelihood of inclusion in the final chain and earning rewards.
+
+- **Missed Events and Penalties**: In Ethereum PoS, validators can be penalized for failing to participate in the consensus process. If your network connection is slow or unstable, it could cause missed attestations or proposals, resulting in penalties and reduced rewards.
+  Network Resilience: A reliable internet connection with low latency helps your validator node stay connected and recover quickly from temporary network outages. Strength reduces the risk of being isolated from the rest of the network and missing important consensus events.
+
+### What are the minimal network speeds to run a node?
+
+Sufficient network speeds and low latency are essential for running an Ethereum PoS validator node at home. It ensures optimal performance, maximizes reward potential, and minimizes the risk of penalties or missed attestations.
+
+A minimum download speed of `10-15 Mbps` is recommended, with higher rates being more desirable for faster blockchain synchronization and data transfer. Upload speeds should be at least `2-4 Mbps` to ensure your attestations and proposals can be sent to the network quickly. Regarding latency, a connection below `30-100 ms` is ideal for guaranteeing timely participation in the consensus process and reducing the risk of missing attestations or proposals.
+
```

Comment:
Line 10, you said that speed doesn't matter as long as the validator answer in 12s , but in this paragraph (and also line 39), you state that speed is important.

Reply:
Every 12 seconds is the new block in which the network has to receive the answer. I'm going to update the description...

> In summary, while the block time of 12 seconds defines the average time between blocks, having good latency remains important for timely synchronization, transaction propagation, and network participation.